docs/contributing/submit-to-creator-showcase.md

```diff
@@ -1,33 +1,33 @@
 ---
-title: Submit to Creator Showcase
+title: Submeter ao Mostruário de Criadores
```

Comment:
Traduzi a página submit-to-plugin-library e utilizei a palavra "enviar" ao invés de "submeter". O que acha de usarmos "enviar"?